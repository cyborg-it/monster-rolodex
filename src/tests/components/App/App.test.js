import renderer from 'react-test-renderer';
import App from '../../../App';

describe('snapshot test suites for App component', () => {
    it('render App component', () => {
        const tree = renderer.create(<App />).toJSON();
        expect(tree).toMatchSnapshot()
    });
})