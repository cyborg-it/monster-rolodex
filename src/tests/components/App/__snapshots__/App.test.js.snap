// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test suites for App component render App component 1`] = `
<div
  className="App"
>
  <h1>
    Monster rolodex
  </h1>
  <input
    className="search"
    onChange={[Function]}
    placeholder="search monsters"
    type="search"
  />
  <div
    className="card-list"
  />
</div>
`;
